lisa/features/network_interface.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Type
+
+from lisa import schema
+from lisa.feature import Feature
+from lisa.schema import NetworkInterfaceOptionSettings
+
+
+class NetworkInterface(Feature):
```

Comment:
It looks there is no difference from Azure network interface. So it doesn't need to have an extension. The base feature can be used directly in AzurePlatform. We can add this level, when it's necessary.